### ISSUE TITLE
NPE crash

### ISSUE THREAD

**Issue by purpleposeidon** [The-Fireplace/Overlord#135]
I had just used the horn while the skeles were maybe 30 blocks away. Certainly not a consistent thing.
```
java.lang.NullPointerException: Ticking entity
	at net.minecraft.entity.ai.EntityLookHelper.func_75651_a(SourceFile:19)
	at net.minecraft.entity.ai.EntityAIAttackMelee.func_75246_d(EntityAIAttackMelee.java:123)
	at net.minecraft.entity.ai.EntityAITasks.func_75774_a(SourceFile:129)
	at net.minecraft.entity.EntityLiving.func_70626_be(EntityLiving.java:763)
	at net.minecraft.entity.EntityLivingBase.func_70636_d(EntityLivingBase.java:2350)
	at net.minecraft.entity.EntityLiving.func_70636_d(EntityLiving.java:577)
	at the_fireplace.overlord.entity.EntityArmyMember.func_70636_d(EntityArmyMember.java:200)
	at the_fireplace.overlord.entity.EntitySkeletonWarrior.func_70636_d(EntitySkeletonWarrior.java:423)
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:2170)
	at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:295)
	at net.minecraft.world.World.func_72866_a(World.java:1990)
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:832)
	at net.minecraft.world.World.func_72870_g(World.java:1952)
	at net.minecraft.world.World.func_72939_s(World.java:1756)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:613)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:765)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:666)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:185)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:524)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by The-Fireplace** [comment]
Thanks for letting me know. I'll check into it when I get the chance, it'll
probably be a week or two before I can do anything about it.

**Comment by Sunconure11** [comment]
@The-Fireplace I found something similar, with the aid of VanillaFix.

https://paste.dimdev.org/feronayofe.mccrash

It appears to be connected to HWYLA in some way.

**Comment by The-Fireplace** [comment]
Yeah, I'm betting the crash from @Sunconure11 happens when you look at a skeleton warrior
maker when you have waila enabled. I'll check in to it when I get the
chance.

**Comment by The-Fireplace** [comment]
@Sunconure11 Your issue should be fixed in 2.4.13

**Comment by The-Fireplace** [comment]
@purpleposeidon Am I correct in assuming at least one of your skeletons was in combat when the crash occurred? Note to self, see if waiting until the AI finishes executing before teleporting skeletons fixes the issue.
